src/components/AppContent.vue

```diff
@@ -57,9 +57,12 @@ export default {
   },
   methods: {
     onClick (event) {
-      // click
+      // click event should be not stopped if content edition is enabled
       if (!this.contentEdition) return
-      if (event) event.preventDefault()
+      if (event) {
+        event.stopPropagation()
```

Comment:
Why do you state that event `should be not stopped` while you `stopPropagation` to parents below?

Reply:
Good catch, it is a typo. I meant `event should not be propagated`.